### PR TITLE
SUS-3068 query wikicities rather than wikicities_c1

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SUS-3068

Get rid of
```
SELECT 
log_id,user_name  FROM `logging`,`wikicities_c1`.`user`  WHERE log_id BETWEEN 242701 AND 242692 AND log_user = user_id
```

in favour of fetching users from `wikicities` with `User::whoAre() method introduced in https://github.com/Wikia/app/pull/14034

https://github.com/Wikia/app/pull/14034 is a dependency for this one.

### CODE REVIEW

**Review by mszabo-wikia — APPROVED** [review]
Are we sure this script is still needed?

**Comment by ghost** [comment]
@TK-999 — I've asked that question in the ticket.  It might not be needed once we remove `log_user_text` column.

File: maintenance/populateLogUsertext.php [hunk 1]
```diff
@@ -53,16 +53,34 @@ protected function doDBUpdates() {
...
+
+			$ids = [];
+
+			foreach ( $res as $row ) {
+				$ids[] = $row->log_user;
+			}
+
+			$res->rewind();
+
+			$users = User::whoAre( $ids );
```
Comment (macbre) [inline maintenance/populateLogUsertext.php:74]: Will `$ids` have unique values or should we first call `array_unique` over them?

Comment (ghost) [reply maintenance/populateLogUsertext.php:74]: `User::whoAre()` uses MySQL's `DISTINCT`.  We can consider using `array_unique` inside the method.  We are discussing that in the relevant pull request.

**Pull request merged**